OvmfPkg/OvmfPkg.dec

```diff
@@ -309,3 +309,4 @@
   #  firmware contains a CSM (Compatibility Support Module).
   #
   gUefiOvmfPkgTokenSpaceGuid.PcdCsmEnable|FALSE|BOOLEAN|0x35
+  gEfiMdeModulePkgTokenSpaceGuid.PcdCreatePreInstalledBootOptions|TRUE|BOOLEAN|0x0001007a

```

Comment:
@filipleple It is already defined in `./MdeModulePkg.dec` - I'm not sure if we should define it also here. What is the problem if you do not define it here?

Reply:
Turned out to be overkill, removed in commit [bf80838](https://github.com/Dasharo/edk2/commit/bf80838ddff98bad5596cd1a3d43e254bb21313e)

Reply:
Thanks